spec.html

```diff
@@ -12642,9 +12646,11 @@ <h1>Runtime Semantics: ArrayAccumulation</h1>
         <emu-alg>
           1. Let _len_ be _nextIndex_ + 1.
           1. Perform ? Set(_array_, *"length"*, _len_, *true*).
-          1. NOTE: The above Set throws if _len_ exceeds 2<sup>32</sup>-1.
           1. Return _len_.
         </emu-alg>
+        <emu-note>
+          <p>The above Set throws if _len_ exceeds 2<sup>32</sup>-1.</p>
```

Comment:
I think this one might be the most subjective. Technically it describes a single line so I'm happy to leave it as-is, but then the algorithm as a whole is tiny and I think endnotes tend to be cleaner than inline notes (though part of that is because we currently lack special styling for inline notes).